src/js/view/view.js

```diff
@@ -249,11 +249,21 @@ define([
             }
 
             _container = _playerElement.getElementsByClassName('jw-main')[0];
```

Comment:
Plugins will go into .jw-overlays, (this is annotated in lucidcharts using
a comment)
On May 5, 2015 11:58 AM, "jw-kaurand" <notifications@github.com> wrote:

> In src/js/view/view.js
> <https://github.com/jwplayer/jwplayer/pull/426#discussion_r29683221>:
>
> > @@ -249,11 +249,21 @@ define([
> >              }
> >
> >              _container = _playerElement.getElementsByClassName('jw-main')[0];
>
> Seems like this is causing some issues with
> jwplayer-commercial/test/vast-linear.html due to it looking for the jw-main
> class
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jwplayer/jwplayer/pull/426/files#r29683221>.
>